src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py

```diff
@@ -293,6 +293,21 @@
                 "Table of alpha values for each month. "
                 "Required if Use Monthly Alpha Table is selected."),
             "name": gettext("monthly alpha table")
+        },
+        "algorithm": {
+            "type": "option_string",
+            "options": {
+                "D8": {
+                    "display_name": gettext("D8"),
+                    "description": "D8 flow direction"
+                },
+                "MFD": {
+                    "display_name": gettext("MFD"),
+                    "description": "Multiple flow direction"
+                }
+            },
+            "about": gettext("Flow direction algorithm to use."),
+            "name": gettext("flow direction algorithm")
```

Comment:
Since we have this defined in 3 models, would it be worth abstracting the options out into an object in `spec_utils`?

Reply:
Oh good idea! I moved it to `spec_utils` and also renamed the key to `flow_dir_algorithm` to be more specific